x86_64-darwin failure is:

```
53% tests passed, 57 tests failed out of 120

Total Test time (real) =  38.41 sec

The following tests FAILED:
          3 - testXmlParserCamera (Subprocess aborted)
         14 - perfMatrixMultiplication (SEGFAULT)
         19 - testHomogeneousMatrix (SEGFAULT)
         21 - testKalmanAcceleration (SEGFAULT)
         22 - testKalmanVelocity (SEGFAULT)
         23 - testLineFitting (SEGFAULT)
         25 - testMathUtils (SEGFAULT)
         26 - testMatrix (SEGFAULT)
         27 - testMatrixConditionNumber (Failed)
         29 - testMatrixDeterminant (SEGFAULT)
         32 - testMatrixInverse (SEGFAULT)
         33 - testMatrixPseudoInverse (SEGFAULT)
         39 - testRobust (Failed)
         41 - testRotation2 (SEGFAULT)
         43 - testSvd (SEGFAULT)
         46 - testXmlParserHomogeneousMatrix (Subprocess aborted)
         54 - testXmlParserRectOriented (Subprocess aborted)
         67 - testJsonArgumentParser (Failed)
         70 - testNurbs (SEGFAULT)
         73 - testFeature (SEGFAULT)
         74 - testFeatureMoment (SEGFAULT)
         75 - testFeatureSegment (SEGFAULT)
         77 - testFindMatch (SEGFAULT)
         78 - testPose (SEGFAULT)
         79 - testPoseFeatures (SEGFAULT)
         80 - testPoseRansac (SEGFAULT)
         82 - servoSimu4Points (SEGFAULT)
         83 - servoSimuCylinder (SEGFAULT)
         84 - servoSimuSphere (SEGFAULT)
         85 - calibrate-hand-eye (SEGFAULT)
         86 - homographyHartleyDLT2DObject (Failed)
         87 - homographyHLM2DObject (Failed)
         88 - homographyHLM3DObject (SEGFAULT)
         89 - homographyRansac2DObject (SEGFAULT)
         92 - quadprog_eq (SEGFAULT)
         93 - quadprog (SEGFAULT)
         97 - servoSimuAfma6FourPoints2DCamVelocity (SEGFAULT)
         98 - servoSimu3D_cdMc_CamVelocity (SEGFAULT)
        100 - servoSimu3D_cMcd_CamVelocity (SEGFAULT)
        102 - servoSimuCircle2DCamVelocity (SEGFAULT)
        103 - servoSimuCircle2DCamVelocityDisplay (SEGFAULT)
        104 - servoSimuCylinder2DCamVelocityDisplay (SEGFAULT)
        105 - servoSimuFourPoints2DCamVelocity (SEGFAULT)
        106 - servoSimuFourPoints2DCamVelocityDisplay (SEGFAULT)
        107 - servoSimuLine2DCamVelocityDisplay (SEGFAULT)
        108 - servoSimuPoint2DCamVelocity1 (SEGFAULT)
        109 - servoSimuPoint2DCamVelocity2 (SEGFAULT)
        110 - servoSimuPoint2DCamVelocity3 (SEGFAULT)
        111 - servoSimuPoint2DhalfCamVelocity1 (SEGFAULT)
        112 - servoSimuPoint2DhalfCamVelocity2 (SEGFAULT)
        113 - servoSimuPoint2DhalfCamVelocity3 (SEGFAULT)
        114 - servoSimuPoint3DCamVelocity (SEGFAULT)
        115 - servoSimuSphere2DCamVelocity (SEGFAULT)
        116 - servoSimuSphere2DCamVelocityDisplay (SEGFAULT)
        117 - servoSimuSphere2DCamVelocityDisplaySecondaryTask (SEGFAULT)
        118 - servoSimuThetaUCamVelocity (SEGFAULT)
        119 - servoSimuViper850FourPoints2DCamVelocity (SEGFAULT)
Errors while running CTest
make: *** [Makefile:71: test] Error 8

```

--- a/pkgs/by-name/vi/visp/package.nix
+++ b/pkgs/by-name/vi/visp/package.nix
@@ -102,6 +102,8 @@ stdenv.mkDerivation (finalAttrs: {
 
   meta = {
     description = "Open Source Visual Servoing Platform";
+    # ref. https://github.com/lagadic/visp/pull/1658
+    broken = stdenv.hostPlatform.system == "x86_64-darwin";
     homepage = "https://visp.inria.fr";
     changelog = "https://github.com/lagadic/visp/blob/v${finalAttrs.version}/ChangeLog.txt";
     license = lib.licenses.gpl2Plus;

